### ISSUE TITLE
安装platon的几处问题

### ISSUE THREAD

**Issue by szfdiwang** [PlatONnetwork/Docs#165]
[源码安装PlatON文档中](https://devdocs.platon.network/docs/zh-CN/Install_PlatON/)
在执行到编译时, 出现几个问题
> cd $GOPATH/src/github.com/PlatONnetwork/PlatON-Go
go run build/ci.go install ./cmd/platon
go run build/ci.go install ./cmd/platonkey

1. 无法执行编译,  go第三方包需要安装 ,  需要先执行 go mod download .
2. go mod download 后, 依旧无效 需要先执行 go mod tidy, 再进行 go mod download

3. 完成以上步骤后, 进行编译 , 编译过程后报错, 且只生成了 ./cmd/platon 文件 并没有如文档所说生成 ./cmd/platonkey 文件
![1636517776(1)](https://user-images.githubusercontent.com/25865789/141048680-008e5eff-415e-4541-832d-bab008c231fd.png)
![image](https://user-images.githubusercontent.com/25865789/141048875-534568bd-1a61-472f-8967-e0c90bd90f80.png)
4. 请给予帮助

**Comment by kaf-lamed-beyt** [comment]
Hi @biganxin do you mind me working on this?

If yes, can you put me through what I can do?

**Comment by biganxin** [comment]
@Caleb335 Yes, you can do it, mainly to optimize the documentation and solve the problems encountered by the users

**Comment by biganxin** [comment]
@benbaley

**Comment by kaf-lamed-beyt** [comment]
@biganxin, should I still work on this?

**Comment by biganxin** [comment]
@Caleb335 Yes, it's open
